client/src/views/TrackViewer/Layers.vue

```diff
@@ -202,14 +202,8 @@ export default defineComponent({
 
     editAnnotationLayer.$on('update:geojson',
       (data: GeoJSON.Feature<GeoJSON.Polygon>) => {
-        const track = props.trackMap.get(props.selectedTrackId.value);
-        if (track) {
-          const bounds = geojsonToBound(data);
-          track.setFeature({
-            frame: frameNumber.value,
-            bounds,
-          });
-        }
```

Comment:
(1) After reading through `useModeManager`, I'm not sure I see the value o pulling all those handlers into a separate file.

Aside from `handleUpdateRectBounds`, it's a collection of very simple wrapper functions that are tightly coupled with the dependencies of `Viewer`.  I think the intention of a `use*` composition function is to factor out some easily separable feature that the viewer relies on.  `useModeManager` depends on everything, including template refs, which makes me a little uncomfortable.